.github/workflows/unit.yml

```diff
@@ -0,0 +1,35 @@
+on: [push, pull_request]
+
+jobs:
+  unit:
+    strategy:
+      matrix:
+        os: [ "ubuntu", "windows", "macos" ]
+        go: [ "1.14", "1.15" ]
+    runs-on: ${{ matrix.os }}-latest
+    name: Unit tests (${{ matrix.os}}, Go ${{ matrix.go }})
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with: 
+          go-version: ${{ matrix.go }}
+      - run: go version
+      - name: Install test tools
+        run: |
+          go get golang.org/x/tools/cmd/cover
+          go get github.com/onsi/ginkgo/ginkgo
+          go get github.com/onsi/gomega
```

Comment:
assuming these tools are tracked in your go.mod, it would be best to use `go install`, as `go get` is increasingly geared towards updating/adding dependencies to go.mod.

Reply:
We need the `ginkgo` command line utility, so `go install` won't work. We don't need the cover tool and gomega though.

Reply:
Why wouldn't `go install` work? As long as these module deps are in go.mod, `go install` is better than `go get`, because `go get` is really meant for updating go.mod

Reply:
`go install` doesn't work, as we need the `ginkgo` binary. `go install github.com/onsi/ginkgo/ginkgo` works though.

Reply:
Is that better?

Reply:
Yes, that's better :) I did mean replacing `go get github.com/onsi/ginkgo/ginkgo` with `go install github.com/onsi/ginkgo/ginkgo`, perhaps I wasn't very clear.